README.md

```diff
@@ -39,14 +39,37 @@ Here's the first example to get you started: a counter that can go up or down. Y
     <script type="module">
       import { h, app } from "https://unpkg.com/hyperapp"
 
+      const cloneObject = function (obj) {
+        return JSON.parse(JSON.stringify(obj))
+      }
+
+      const decrement = function (state) {
+        return function () {
+          const newState = cloneObject(state)
+          newState.counter -= 1
+          return newState
+        }
+      }
```

Comment:
This is how you'd write this action:

```js
const decrement = state => ({ ...state, counter: state.counter - 1 })
```

Reply:
This is very confusing to somebody who doesn't know all of the tricks/syntactic sugar of ES6. I get that you don't really care about those people (and you shouldn't have to)

but... say a newcomer from Java/Scala wants to come dive into your project. My example is much more verbose/clear on what is happening.

Reply:
I don't know if you're being serious or trolling me. 😉  

Your example creates a closure for no reason. Then clones the state inefficiently, mutates it, and finally returns it. Unless you're completely new to programming, `state => state - 1` should be readable enough.

Reply:
Examples are supposed to show case real world usage.

In what real world usage is the entire state for a component simply an `integer`? An object is much more realistic.

Also, arrow functions didn't exist in JavaScript until semi-recently (and you won't find them in other languages).

I'm not trolling you (and I'm not questioning how long you've been programming, like you are to me). I'm trying to collaborate on making a real world example that doesn't use clever tricks.

Reply:
True, no real world example will just have a number as the state. However, I disagree that all examples are for showcasing real-world use cases. The point of this first example is to give an idea at a quick glance, how the framework works.

There are plenty of other examples (official and community) that are more realistic.

One thing I would suggest though, @jorgebucaran, is linking to the tutorial and examples from the readme.

Reply:
> There are plenty of other examples (official and community) that are more realistic.

Why have an unrealistic example in the README (one of the first things people see when they land on your project)?

Reply:
@zaceno Definitely a good idea! I just haven't found a way to do it that stays in harmony with the rest of the file, but I'll see what I can do! 😄